playground/infrastructure/cloudbuild/cloudbuild_playground_infra_part2.yaml

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+steps:
+  - name: ubuntu
+    args:
+      - '-c'
+      - >-
+        apt update > /dev/null
+
+        apt-get install -y wget unzip software-properties-common git curl
+        apt-transport-https ca-certificates gnupg jq > /dev/null
+
+        echo "deb https://packages.cloud.google.com/apt cloud-sdk main" | tee -a
+        /etc/apt/sources.list.d/google-cloud-sdk.list
+
+        curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add
+        -
+
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+
+        add-apt-repository "deb [arch=amd64]
+        https://download.docker.com/linux/ubuntu focal stable" > /dev/null
+
+        apt install -y docker-ce > /dev/null
+
+        wget
+        https://releases.hashicorp.com/terraform/1.4.2/terraform_1.4.2_linux_amd64.zip 
+
+        unzip terraform_1.4.2_linux_amd64.zip
+
+        mv terraform /usr/local/bin/terraform
+
+        terraform --version
+
+        apt update -y > /dev/null && apt install -y google-cloud-sdk
+        openjdk-11-jdk kubectl > /dev/null 
+
+        curl -fsSLo get_helm.sh
+        https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 >
+        /dev/null
+
+        chmod +x get_helm.sh && ./get_helm.sh > /dev/null
+
+        apt-get install google-cloud-sdk-gke-gcloud-auth-plugin
+
+        git clone --branch merged_pg_gke_deployment
+        https://github.com/akvelon/beam.git
```

Comment:
I don't like that we are hardcoding the repo and branch name

Reply:
It's fine. But it should be apache, not akvelon